bio/quast/wrapper.py

```
@@ -67,9 +71,62 @@
 
-output_dir = os.path.commonpath(snakemake.output)
+with tempfile.TemporaryDirectory() as tmpdir:
+    shell(
+        "quast --threads {snakemake.threads} {ref} {gff} {pe1} {pe2} {pe12} {mp1} {mp2} {mp12} {single} {pacbio} {nanopore} {ref_bam} {ref_sam} {bam} {sam} {sv_bedpe} {extra} -o {tmpdir} {snakemake.input.fasta} {log}"
+    )
 
+    ### Copy files to final destination
+    def save_output(src, dst, wd=Path(".")):
+        if not dst:
+            return 0
+        dest = wd / dst
```

Ruff

Local variable `dest` is assigned to but never used

Remove assignment to unused variable `dest`

(F841)